apps/web/src/components/file-browser/DetailsDialog.tsx

```
@@ -0,0 +1,305 @@
+      </DetailRow>
+
+      <DetailRow label="Type">
+        <span className="details-type-badge details-type-badge--file">[FILE]</span>
+      </DetailRow>
+
+      <DetailRow label="Size">
+        <span className="details-value">{formatBytes(item.size)}</span>
+      </DetailRow>
+
+      {/* Crypto section */}
+      <div className="details-section-header">// encryption</div>
```

Biome (2.3.13)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)